### DISCUSSION TITLE
Docstrings in dpt_save_data

### DISCUSSION THREAD

**Discussion by MohanNagella** [upb-lea/transistordatabase#54] [General]
Dear @henninghst ,

There are unwanted docstrings in the function **_dpt_save_data_** . Kindly review and remove them :)
And also the variables off_vds_limit, off_is_limit, on_vds_limit, on_is_limit if possible try to intialize them with None before,  rather than intializing them for the first time within the if blocks

**Comment by gituser789** [comment]
Done
